### DISCUSSION TITLE
New Project Brainstorm

### DISCUSSION THREAD

**Discussion by gilbertogne** [ffcoders/ffcoders-page#11] [Ideas]
I have 2 ideas for my new project. 

1 - A web application that helps me to manage my squad in the Hattrick's game.
2 - A web application that I can control games I want to play, to buy, .. and manage wishlist for playing.

My objective is join an API to a web application like vue.js maybe.

What do you think , guys?

**Comment by alextcf** [comment]
When you say it helps manage your team in the game, what do you mean? Is it some integration with the game itself?

**Reply by gilbertogne** [reply]
There are some applications that can enable this conection with the game, but I was thinking about a stand alone app to add those data to my application.

**Reply by alextcf** [reply]
Ok, so we need to organize the desired functionalities. This way we can separate the tasks for each person. I have no experience with prototypes, but we can ask Cris if he creates any and we can see more clearly what we need. Do the following: write down the functionalities you want in a list and with that we can imagine them to generate a prototype.

**Reply by gilbertogne** [reply]
It's just a project that I was thinking for me. We don't need to follow this idea if you have another good one.
